### ISSUE TITLE
Android crash importantForAccessibility

### ISSUE THREAD

**Issue by arthurfauq** [peacechen/react-native-modal-selector#141]
Hi,
The component is working great except on Android device where my app crashes when the picker opens with the error:

![image](https://user-images.githubusercontent.com/17045144/78249564-d999df80-74ee-11ea-80c4-54573958744f.png)

It seems to come from this line in the `renderOption` method:
```
importantForAccessibility={isFirstItem}
```
which receives a boolean instead of one of the accepted values.

Versions:
```
"react-native": "0.62.0",
"react-native-modal-selector": "^2.0.0"
```

Thanks in advance !

**Comment by vicdup** [comment]
See https://github.com/peacechen/react-native-modal-selector/pull/142

**Comment by peacechen** [comment]
@vicdup 's PR has been merged.  Before publishing a new version, please validate it on older versions of RN to ensure that it doesn't break them.

Use this in your `package.json` to pull directly from the repo:
```
dependencies: {
  "react-native-modal-selector": "https://github.com/peacechen/react-native-modal-selector.git",
}
```

**Comment by nibcortes04** [comment]
> @vicdup 's PR has been merged. Before I publish a new version, please validate it on older versions of RN to ensure that it doesn't break them.
> 
> Use this in your `package.json` to pull directly from the repo:
> 
> ```
> dependencies: {
>   "react-native-modal-selector": "https://github.com/peacechen/react-native-modal-selector.git",
> }
> ```

Hola, tengo este mismo error en este momento, agregue en las dependencias lo que dices pero sigue fallando.

**Comment by peacechen** [comment]
@vicdup @arthurfauq 
Have you had a chance to test this on current and older versions of RN?

**Comment by nibcortes04** [comment]
> @vicdup @arthurfauq
> ¿Has tenido la oportunidad de probar esto en versiones actuales y anteriores de RN?

Si, anteriormente lo tenia, lo monte de nuevo en una nueva y no funciona.

**Comment by peacechen** [comment]
Mi Espanol es no beuno.  English please.

Have you tried to pull directly from this repo as noted above https://github.com/peacechen/react-native-modal-selector/issues/141#issuecomment-608023186

Double check that the code in `node_modules/react-native-modal-selector/index.js` is the most recent.  This line should match:
https://github.com/peacechen/react-native-modal-selector/blob/master/index.js#L236

Clear the metro and watchman caches to make sure the bundler fetches the updated code.
Linux/OSX:
https://medium.com/@abhisheknalwaya/how-to-clear-react-native-cache-c435c258834e

Windows:
https://mikebridge.github.io/post/clear-react-native-caches/

**Comment by nibcortes04** [comment]
> Mi Espanol es no beuno. English please.
> 
> Have you tried to pull directly from this repo as noted above [#141 (comment)](https://github.com/peacechen/react-native-modal-selector/issues/141#issuecomment-608023186)
> 
> Double check that the code in `node_modules/react-native-modal-selector/index.js` is the most recent. This line should match:
> https://github.com/peacechen/react-native-modal-selector/blob/master/index.js#L236
> 
> Clear the metro and watchman caches to make sure the bundler fetches the updated code.
> Linux/OSX:
> https://medium.com/@abhisheknalwaya/how-to-clear-react-native-cache-c435c258834e
> 
> Windows:
> https://mikebridge.github.io/post/clear-react-native-caches/

Perfecto, muchas gracias.

**Comment by nochkin** [comment]
> Double check that the code in `node_modules/react-native-modal-selector/index.js` is the most recent. This line should match:
> https://github.com/peacechen/react-native-modal-selector/blob/master/index.js#L236

Tested fine for me. Thank you.

**Comment by peacechen** [comment]
Published in 2.0.1
